packages/palette_generator/test/disabled_canvaskit_test.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: avoid_print
+
+// TODO(dit): Delete this file, https://github.com/flutter/flutter/issues/151498
```

Comment:
Do we need a whole placeholder file, rather than just doing [this](https://github.com/flutter/packages/blob/main/packages/pigeon/dart_test.yaml) with a comment about why?

Reply:
Nope, I just wasn't aware of that config file!